packages/core/src/awsService/cloudWatchLogs/activation.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import * as vscode from 'vscode'
-import { CLOUDWATCH_LOGS_SCHEME } from '../../shared/constants'
+import { CLOUDWATCH_LOGS_LIVETAIL_SCHEME, CLOUDWATCH_LOGS_SCHEME } from '../../shared/constants'
```

Comment:
please rename `CLOUDWATCH_LOGS_LIVETAIL_SCHEME` to conform to the lint rule. `CLOUDWATCH_LOGS_SCHEME` was marked as "ignore lint rule" as a temporary workaround, not something to continue when new symbols are introduced.

Reply:
I will rename, didn't fully realize the linter suppression,  was just following precedent. I can update the variable naming. 

The reason I added a new scheme is because I am not leveraging the existing document and code lens providers. The logic for fetching logs, and the supplied codeLenses for LiveTail are different than those in the existing providers. I didn't know query param would be an option, but since the providers are tied to the scheme itself, I'm not sure if a query param does what I was intending for by having separate providers. Let me know if that can be done!

I could use a query param and a shared scheme, but then I assume that each time one of the providers run, we'd need to inspect the URI and have a branching logic to handle a LiveTail case or not. In that case I'd prefer to just handle these cases in separate classes.

Reply:
>  I am not leveraging the existing document and code lens providers

I believe the existing abstraction is generalized to allow different "providers" of CWL events. So it could be re-used. But it may be a judgement call about whether that saves or increases complexity.